src/padchat-manager.ts

```diff
@@ -742,7 +742,7 @@ export class PadchatManager extends PadchatRpc {
     if (!this.cacheRoomRawPayload) {
       throw new Error('cache not inited' )
     }
-    const roomIdList = [...this.cacheRoomRawPayload.keys()]
+    const roomIdList = [...this.cacheRoomRawPayload.keys()].filter(id => !!id)
```

Comment:
Is there possible that `id` is empty?

I think the this.cacheRoomRawPayload.keys() should never return empty keys.

Reply:
Seems no, I will revert this change back.